cmd/ftl/cmd_schema_import.go

```diff
@@ -0,0 +1,220 @@
+		return err
+	}
+
+	prompt, err := i.getPrompt()
+	if err != nil {
+		return err
+	}
+
+	err = query(ctx, prompt)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func query(ctx context.Context, prompt string) error {
+	logger := log.FromContext(ctx)
+
+	logger.Infof("The import schema command relies on the %s AI chat model to translate schemas from other "+
+		"languages into FTL compliant objects in the specified language. Output may vary and results should be inspected "+
+		"for correctness. It is suggested that if the results are not satisfactory, you try again.\n\n", ollamaModel)
+
+	llm, err := ollama.New(ollama.WithModel("llama2"))
+	if err != nil {
+		logger.Errorf(err, "failed to call ollama")
+	}
+
+	completion, err := llm.Call(ctx, fmt.Sprintf("Human: %s \nAssistant:", prompt),
+		llms.WithTemperature(0.8),
+		llms.WithStreamingFunc(func(ctx context.Context, chunk []byte) error {
+			fmt.Print(string(chunk))
+			return nil
+		}),
+	)
+	if err != nil {
+		logger.Errorf(err, "failed to call ollama")
+	}
+
+	_ = completion
+	return nil
+}
+
+func (s *schemaImportCmd) setup(ctx context.Context) error {
+	logger := log.FromContext(ctx)
+
+	nameFlag := fmt.Sprintf("name=^/%s$", ollamaContainerName)
+	output, err := exec.Capture(ctx, ".", "docker", "ps", "-a", "--filter", nameFlag, "--format", "{{.Names}}")
+	if err != nil {
+		logger.Errorf(err, "%s", output)
+		return err
+	}
+
+	port := fmt.Sprintf("%d", s.OllamaPort)
+
+	if len(output) == 0 {
+		logger.Infof("Creating docker container '%s' for ollama", ollamaContainerName)
+
+		// check if port s.OllamaPort is already in use
+		_, err := exec.Capture(ctx, ".", "sh", "-c", fmt.Sprintf("lsof -i:%d", s.OllamaPort))
+		if err == nil {
+			return fmt.Errorf("port %d is already in use", s.OllamaPort)
+		}
+
+		err = exec.Command(ctx, logger.GetLevel(), "./", "docker", "run",
+			"-d", // run detached so we can follow with other commands
+			"-v", ollamaVolume,
+			"-p", fmt.Sprintf("%s:11434", port),
+			"--name", ollamaContainerName,
+			"ollama/ollama",
+		).Run()
+		if err != nil {
+			return err
+		}
+	} else {
+		// Start the existing container
+		_, err = exec.Capture(ctx, ".", "docker", "start", ollamaContainerName)
+		if err != nil {
+			return err
+		}
+
+		// Grab the port from the existing container
+		portOutput, err := exec.Capture(ctx, ".", "docker", "port", ollamaContainerName, "11434/tcp")
+		if err != nil {
+			logger.Errorf(err, "%s", portOutput)
+			return err
+		}
+		port = slices.Reduce(strings.Split(string(portOutput), "\n"), "", func(port string, line string) string {
+			if parts := strings.Split(line, ":"); len(parts) == 2 {
+				return parts[1]
+			}
+			return port
+		})
+	}
+
+	// Initialize Ollama
+	err = exec.Command(ctx, logger.GetLevel(), "./", "docker", "exec",
+		ollamaContainerName,
+		"ollama",
+		"run", ollamaModel,
+	).Run()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func createInputString(dir string) (string, error) {
```

Comment:
I wonder if we should return the types of the language here, to improve the prompt a bit? ie. so we can tell the LLM what the input language is? Or maybe it's so generalised that it just figures it out on its own?